python/tvm/auto_scheduler/measure_record.py

```diff
@@ -45,6 +45,11 @@ class RecordToFile(MeasureCallback):
     """
 
     def __init__(self, filename):
+        dirname = os.path.dirname(filename)
+        if dirname == "":
+            dirname = "."
```

Comment:
Better not to use ".", use `os.path.abspath` to get the absolute path.

Reply:
changed